File: CS/XPO_with_Web Service/App_Code/MyWebDataStore.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Web.Services;


[WebServiceBinding(Namespace = WebServiceAttribute.DefaultNamespace)]
class MyWebDataStore : DevExpress.Xpo.WebServiceDataStore {
    public MyWebDataStore(string url)
        : base(url, DevExpress.Xpo.DB.AutoCreateOption.SchemaAlreadyExists) {
    }
}

File: VB/XPO_with_Web Service/App_Code/MyWebDataStore.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.Web.Services

' TODO: change MyWebDataSource namespace to match the published service's namespace
<WebServiceBinding(Namespace := WebServiceAttribute.DefaultNamespace)> _
Friend Class MyWebDataStore
	Inherits DevExpress.Xpo.WebServiceDataStore
	Public Sub New(ByVal url As String)
		MyBase.New(url, DevExpress.Xpo.DB.AutoCreateOption.SchemaAlreadyExists)
	End Sub
End Class

